src/sentry/middleware/user.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+from django.utils import timezone
+from datetime import timedelta
+import inspect
+
+
+class UserActiveMiddleware(object):
+    allowed_paths = (
+        'sentry.api.endpoints',
+        'sentry.web.frontend',
+    )
+
+    def process_view(self, request, view_func, view_args, view_kwargs):
+        if not request.user.is_authenticated():
+            return
+
+        now = timezone.now()
+        freq = timedelta(minutes=60)
+        last_active = request.user.last_active
+
+        if last_active and freq > (now - last_active):
+            return
+
+        request.user.last_active = now
+        request.user.save()
+
+        view = view_func
```

Comment:
This was to limit `allowed_paths`, to avoid event or commit ingestion endpoints, which aren't reflective of an active user. However...that might actually just be caught by the `user.is_authenticated`, given that there's no user in those requests.

Try testing those out locally, see what happened.